tests/mail/tests.py

```diff
@@ -574,6 +574,9 @@ def test_dont_base64_encode(self):
         s = msg.message().as_bytes()
         self.assertNotIn(b'Content-Transfer-Encoding: quoted-printable', s)
         self.assertIn(b'Content-Transfer-Encoding: 8bit', s)
+        s = msg.message().as_string()
+        self.assertNotIn(str('Content-Transfer-Encoding: quoted-printable'), s)
```

Comment:
Is it possible that the header can be duplicated? If not, I don't see the value in the NotIn assertions (perhaps you can explain).